### ISSUE TITLE
“App Settings” item doesn’t use a link cursor, can’t be navigated to with keyboard

### ISSUE THREAD

**Issue by ticky** [glitch-soc/mastodon#375]
The “App Settings” item in the “Getting Started” sidebar is a link with no href attribute and thus doesn’t innately use the link cursor or handle keyboard access like its siblings.

It should probably have a `href="#"` and a `preventDefault` on its `click` handler to work around this.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] ~This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).~ I’m on awoo dot space and I couldn’t get this repo running so I can’t actually verify that this happens on glitch itself? I’m exhausted

**Comment by beatrix-bitrot** [comment]
!

is this serious?

**Comment by ticky** [comment]
- **Am I joking?** No.
- **Does this prevent most people from using the thing?** Not really but it means it requires using the mouse specifically and also doesn’t behave consistently.

**Comment by beatrix-bitrot** [comment]
ok, cool thanks. i will assign it to @marrus-sh though it may be a while before it gets fixed.

**Comment by marrus-sh** [comment]
It does use `role=button` and `tabIndex=0` so it should be accessible to screenreaders and tab navigation—so I'm *pretty sure* (correct me if I'm wrong) this is a purely visual problem and can be fixed with CSS.

I'd rather not add an `href` that doesn't do anything because I don't want it to be misinterpreted by the browser as a link when it's actually technically a button. (That would put the URL in the bottom-left corner of the UI too probably right? At least on firefox)

**Comment by ticky** [comment]
You can _tab_ to it, but you can’t _activate_ it with the keyboard because fundamentally it’s a link with no place to go. Browsers only trigger “click” for keyboard events on _links that go places_ or buttons.

It’s _already_ being misinterpreted. You could make it a button but I don’t think the styles would support that?

**Comment by yipdw** [comment]
Is there a problem with applying `href="#"` for now to work around the immediate problem?
